src/test/cbmc/ForeignItems/fixme_main.rs

```diff
@@ -3,6 +3,8 @@
 //! To run this test, do
 //! rmc fixme_main.rs -- lib.c
 
+// rmc-flags: --c-lib lib.c
```

Comment:
is this still a fixme test?

Reply:
There are two commented out tests below that still don't work:
```
        //assert!(takes_ptr_option(Some(&5)) == 4);
        //assert!(takes_ptr_option(None) == 0);
```

I made a non-fixme one to demonstrate the working functionality of `--c-lib`.